Gems/RemoteTools/Code/Source/RemoteToolsSystemComponent.cpp

```diff
@@ -565,7 +565,7 @@ namespace RemoteTools
         }
 
         // Check if there are any active connections, if not stop the outbox thread
-        bool hasActiveConnection = false;
+        [[maybe_unused]] bool hasActiveConnection = false;
```

Comment:
This looks like temporary testing code to me, nothing is done with this boolean. Can it be removed completely?

Reply:
Looks like the following block was removed after this check for active connection:
```
        if (!hasActiveConnection)
        {
            m_outboxThread->Stop();
            m_outboxThread->Join();
        }
```
It looks like we don't need this check at all

Reply:
This code can be removed.

hasActiveConnection is written to below on line 571, but nothing actually reads it at all.

Reply:
Removed